Content.Shared/Hands/EntitySystems/SharedHandsSystem.Relay.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Content.Shared.Blocking;
+using Content.Shared.Damage;
+using Content.Shared.Hands.Components;
+
+namespace Content.Shared.Hands.EntitySystems;
+
+public abstract partial class SharedHandsSystem : EntitySystem
+{
+    private void InitializeRelay()
+    {
+        SubscribeLocalEvent<SharedHandsComponent, DamageChangedEvent>(RelayHandItemsEvent);
+    }
+
+    private void RelayHandItemsEvent<T>(EntityUid uid, SharedHandsComponent component, T args) where T : EntityEventArgs
+    {
+        var items = EnumerateHeld(uid, component);
+
+        foreach (var item in items)
+        {
+            //A bit singleton but I don't want other destructable items in hand
+            //To be effected
+            if (HasComp<SharedBlockingComponent>(item))
+            {
+                RaiseLocalEvent(item, args, false);
+            }
+        }
+
+    }
```

Comment:
This is pretty dirty, what's the goal here?

Reply:
To pass damage onto the blocking object so that if it has destructible, it can properly break.

Reply:
That didn't seem to work when I tried it.